app.py

```diff
@@ -31,6 +31,38 @@ def launch_request_handler(handler_input: HandlerInput) -> Response:
     return handler_input.response_builder.response
 
 
+@sb.request_handler(can_handle_func=lambda input:
+                    not currently_playing(input) and
+                    is_intent_name('TableNumberIntent')(input))
+def table_number_intent_handler(handler_input):
+    """Handler for Yes Intent, only if the player said yes for
+    a new game.
```

Comment:
These can probably be on the same line right? Check out python docstring conventions https://www.python.org/dev/peps/pep-0257/